src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Custom/ExposureSubmissionDynamicCell.swift

```diff
@@ -21,20 +21,48 @@ import Foundation
 import UIKit
 
 enum ExposureSubmissionDynamicCell {
-	static func stepCell(title: String, description: String?, icon: UIImage?, iconTint: UIColor? = nil, hairline: ExposureSubmissionStepCell.Hairline, bottomSpacing: ExposureSubmissionStepCell.Spacing = .large, action: DynamicAction = .none) -> DynamicCell {
+	static func stepCell(
+		title: String,
+		description: String?,
+		icon: UIImage?,
+		iconTint: UIColor? = nil,
+		hairline: ExposureSubmissionStepCell.Hairline,
+		bottomSpacing: ExposureSubmissionStepCell.Spacing = .large,
+		action: DynamicAction = .none
+	) -> DynamicCell {
 		.identifier(ExposureSubmissionSuccessViewController.CustomCellReuseIdentifiers.stepCell, action: action) { _, cell, _ in
 			guard let cell = cell as? ExposureSubmissionStepCell else { return }
 			cell.configure(title: title, description: description, icon: icon, iconTint: iconTint, hairline: hairline, bottomSpacing: bottomSpacing)
 		}
 	}
 
-	static func stepCell(style: ENAFont, color: UIColor = .enaColor(for: .textPrimary1), title: String, icon: UIImage? = nil, iconAccessibilityLabel: String? = nil, iconTint: UIColor? = nil, hairline: ExposureSubmissionStepCell.Hairline, bottomSpacing: ExposureSubmissionStepCell.Spacing = .large, action: DynamicAction = .none) -> DynamicCell {
+	static func stepCell(
+		style: ENAFont,
+		color: UIColor = .enaColor(for: .textPrimary1),
+		title: String,
+		icon: UIImage? = nil,
+		iconAccessibilityLabel: String? = nil,
+		accessibilityLabel: String? = nil,
+		accessibilityTraits: UIAccessibilityTraits? = nil,
```

Comment:
I think you could write it like this and leave the `if let` out in the assignment.
```suggestion
		accessibilityTraits: UIAccessibilityTraits = [],
```